### PR TITLE
Add food_count to plugin config

### LINKED ISSUE #20: food price not being displayed on prereg page
image

says
"Pay an extra $ for food"

probably means to say:
"Pay an extra $20 for food"

### PR DESCRIPTION
**Pull request by kitsuta**
Also makes the c object actually include the plugin config. Fixes #20.

### CODE REVIEW

**Comment by binary1230** [comment]
this looks good to me, one important note:

Now that this INI setting is in its own plugin, the ```food_price``` INI setting needs to be in plugins/magstock/development.ini, which puppet current doesn't create.

i.e. At the moment, unless/until I add a magstock-specific INI setting, we can't set this value from puppet.  The default value here of 20 is fine for the moment though.

With that in mind: What you created here is totally correct though and the limitation I'm describing is a puppet one, and not a problem with sideboard or the magstock plugin.

See https://github.com/magfest/ubersystem-puppet/issues/53 for more details on the real fix.

TLDR: This is totally fine and correct, keep it this way, and we need to update puppet to handle this instead.

File: magstock/configspec.ini [hunk 1]
```diff
@@ -1,2 +1,2 @@
-
-
+# The cost of a food wristband, which entitles attendees to meals provided by the event.
+food_price = integer(default='20')
```
Comment (EliAndrewC) [inline magstock/configspec.ini]: Minor style note: It would be more idiomatic to say ``integer(default=20)`` without the quotes, though the end result is the same.

**Comment by EliAndrewC** [comment]
+1 (belatedly)
